src/main/java/uk/gov/ons/ctp/integration/rhsvc/service/impl/CaseServiceImpl.java

```diff
@@ -235,7 +235,7 @@ private FulfilmentRequest createFulfilmentRequestPayload(
       if (deliveryChannel == DeliveryChannel.POST) {
         validateContactName(contact);
       }
-      if (product.getCaseTypes().contains(Product.CaseType.HH)) {
+      if (Product.CaseType.HH.name().equals(caseDetails.getCaseType())) {
```

Comment:
Strictly the case.caseType should be compared to CaseType not Product.CaseType. The latter had to be in the Product lib project as it was used by FSDR, and avoided dragging in common framework for them. Not necessary any more as they no longer use - product ref lib was written to be standalone you see. Product.CaseType should really only be used when talking to product lib as that is its lingua. It works though, and the two will hardly drift now.

Reply:
changed to CaseType